pkg/jobrunaggregator/jobrunbigqueryloader/disruption_junit.go

```diff
@@ -1,130 +0,0 @@
-		"ingress-to-console-new-connections":                "Console remains available via cluster frontend ingress using new connections",
-		"ingress-to-console-used-connections":               "Console remains available via cluster frontend ingress using reused connections",
-	}
-
-	e2eBackendNameToTestSubstring = map[string]string{
-		"kube-api-new-connections":         "kube-apiserver-new-connection",
-		"kube-api-reused-connections":      "kube-apiserver-reused-connection should be available",
-		"openshift-api-new-connections":    "openshift-apiserver-new-connection should be available",
-		"openshift-api-reused-connections": "openshift-apiserver-reused-connection should be available",
-		"oauth-api-new-connections":        "oauth-apiserver-new-connection should be available",
-		"oauth-api-reused-connections":     "oauth-apiserver-reused-connection should be available",
-	}
-
-	detectUpgradeOutage = regexp.MustCompile(` unreachable during disruption.*for at least (?P<DisruptionDuration>.*) of `)
-	detectE2EOutage     = regexp.MustCompile(` was failing for (?P<DisruptionDuration>.*) seconds `)
-)
-
-func getServerAvailabilityResultsFromJunit(suites *junit.TestSuites) map[string]jobrunaggregatorlib.AvailabilityResult {
-	availabilityResultsByName := map[string]jobrunaggregatorlib.AvailabilityResult{}
-
-	for _, curr := range suites.Suites {
-		currResults := getServerAvailabilityResultsBySuite(curr)
-		addUnavailability(availabilityResultsByName, currResults)
-	}
-
-	return availabilityResultsByName
-}
-
-func getServerAvailabilityResultsBySuite(suite *junit.TestSuite) map[string]jobrunaggregatorlib.AvailabilityResult {
-	availabilityResultsByName := map[string]jobrunaggregatorlib.AvailabilityResult{}
-
-	for _, curr := range suite.Children {
-		currResults := getServerAvailabilityResultsBySuite(curr)
-		addUnavailability(availabilityResultsByName, currResults)
-	}
-
-	for _, testCase := range suite.TestCases {
-		backendName := ""
-		for currBackendName, testSubstring := range upgradeBackendNameToTestSubstring {
-			if strings.Contains(testCase.Name, testSubstring) {
-				backendName = currBackendName
-				break
-			}
-		}
-		for currBackendName, testSubstring := range e2eBackendNameToTestSubstring {
-			if strings.Contains(testCase.Name, testSubstring) {
-				backendName = currBackendName
-				break
-			}
-		}
-		if len(backendName) == 0 {
-			continue
-		}
-
-		if testCase.FailureOutput != nil {
-			addUnavailabilityForAPIServerTest(availabilityResultsByName, backendName, testCase.FailureOutput.Message)
-			continue
-		}
-
-		// if the test passed and we DO NOT have an entry already, add one
-		if _, ok := availabilityResultsByName[backendName]; !ok {
-			availabilityResultsByName[backendName] = jobrunaggregatorlib.AvailabilityResult{
-				ServerName:         backendName,
-				SecondsUnavailable: 0,
-			}
-		}
```

Comment:
@deads2k does this look like the spot you were remembering where we might be ingesting 0's by default?

I cannot find an equivalent in the code path for ingesting from backend-disruption json files: https://github.com/openshift/ci-tools/blob/master/pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go#L51

Reply:
David confirmed on slack that this, plus ensuring there isn't a console entry in backend-disruption.json files noting 0, should cover it. I confirmed this previously in job runs like https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/27439/pull-ci-openshift-origin-master-e2e-gcp-ovn-upgrade/1573367107456864256/artifacts/e2e-gcp-ovn-upgrade/openshift-e2e-test/artifacts/junit/backend-disruption_20220923-183829.json